tools/goctl/util/ctx/gomod.go

```diff
@@ -81,7 +81,12 @@ func getRealModule(workDir string, execRun execx.RunFunc) (*Module, error) {
 		return nil, err
 	}
 	for _, m := range modules {
-		if strings.HasPrefix(workDir, m.Dir) {
+		mRealDir, err := pathx.ReadLink(m.Dir)
+		if err != nil {
+			continue
```

Comment:
Ignoring errors is a bad programming practice unless you know exactly what you are doing